### ISSUE TITLE
Add proper README

### ISSUE THREAD

**Issue by mikaelbr** [varianter/now-playing#11]
Shold we also do some "productification" at somepoint? Would be cool with some proper documentation, smooth install (don't know how well this works: https://azure.microsoft.com/en-us/blog/deploy-to-azure-button-for-azure-websites-2/) and customization for profile.

Should be a "product" optimized to run on "build screens" maybe?

**Comment by cbrevik** [comment]
Yep definitely! Currently working on ARM templates @ https://github.com/varianter/now-playing/commits/arm-templates 

That should support both "the button", and more traditional CI-systems (wrt. the FAKE-scripts).

**Comment by cbrevik** [comment]
As far as product features; totally agree that we should optimize for the "big screen". But of course be responsive to smaller screens as well.

Customization will also be important. We definitely should allow people to unregister from v1 (or at least set themselves as inactive). Other stuff could follow from that, maybe in a v2?

**Comment by mikaelbr** [comment]
Sounds good to me!
